src/java/picard/sam/revertsam/RevertSam.java

```diff
@@ -359,56 +379,19 @@ public boolean filterOut(final SAMRecord first, final SAMRecord second) {
         return out;
     }
 
-    /**
-     * Takes an individual SAMRecord and applies the set of changes/reversions to it that
-     * have been requested by program level options.
-     */
-    public void revertSamRecord(final SAMRecord rec) {
-        if (RESTORE_ORIGINAL_QUALITIES) {
-            final byte[] oq = rec.getOriginalBaseQualities();
-            if (oq != null) {
-                rec.setBaseQualities(oq);
-                rec.setOriginalBaseQualities(null);
-            }
-        }
+    private String getDefaultExtension(final String input) {
```

Comment:
I think in the usage, since you will want two usage examples anyhow, one for the outputting to one file, and one when outputting per-read-group. @vdauwera does having two examples make sense?